x-pack/plugins/apm/public/application/action_menu/anomaly_detection_setup_link.test.tsx

```diff
@@ -7,7 +7,7 @@
 import React from 'react';
 import { render, fireEvent, waitFor } from '@testing-library/react';
 import { MissingJobsAlert } from './anomaly_detection_setup_link';
-import * as hooks from '../../hooks/useFetcher';
+import * as hooks from '../../hooks/use_fetcher';
```

Comment:
Most changes in this file is simply casing changes, and moving files to other folders.